phpstan.neon.dist

```diff
@@ -25,6 +25,9 @@ parameters:
 			path: %currentWorkingDirectory%/src/Identifier/Normalizer/IntegerDenormalizer.php
 
 		# False positives
+		-
+			message: '#Service "api_platform.iri_converter" is private\.#'
```

Comment:
Should be moved below this line:

https://github.com/api-platform/core/blob/c277aeb8c67bff72188aa65ce896bc3b1f08c468/phpstan.neon.dist#L84